src/components/Modal.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { StyledModal } from "./Modal.styled";
+import PropTypes from "prop-types";
+
+const Modal = ({
+  isVisible,
+  toggleVisibility,
```

Comment:
rename it to `onCancel`, why? because sometimes we want to pass function which is not only responsible for toggling modal but also e.g.: refreshing data, clearing form or so